apps/prairielearn/src/lib/editors.ts

```diff
@@ -888,6 +888,8 @@ export class QuestionAddEditor extends Editor {
     debug('Write info.json with new title and uuid');
     infoJson.title = names.longName;
     infoJson.uuid = this.uuid;
+    // The template question contains tags that shouldn't be copied to the new question.
+    delete infoJson.tags;
```

Comment:
I think the placeholder `"tags"` should be kept...
```suggestion
    infoJson.tags = [];
```

Reply:
IMO it should not be, just as we don't include other "placeholder" values in question `info.json` files.